src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/WristConstants.java

```diff
@@ -28,13 +28,7 @@ public enum PresetPositions {
 
     public static final Rotation2d FORWARD_ANGLE_LIMIT = Rotation2d.fromDegrees(200);
 
-    public static final double RELATIVE_POSITION_CONVERSION_FACTOR = 0.0328;
-
-    public static final double RELATIVE_VELOCITY_CONVERSION_FACTOR = 0.0030332432;
-
-    public static final double ABSOLUTE_POSITION_CONVERSION_FACTOR = 0.00302;
-
-    public static final double ABSOLUTE_VELOCITY_CONVERSION_FACTOR = 0.00302;
+    public static final double CONVERSION_FACTOR = 0.0328;
 
     public static final double kS = 1;
```

Comment:
why is it here? the kS is different for each motor and sub-mechanism thus, it needs to be in each constants file of each subsystem